dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientBase.cs

```diff
@@ -73,7 +73,7 @@ public abstract class ClientBase
         }
 
         Response<Completions>? response = await RunRequestAsync<Response<Completions>?>(
-            async () => await this.Client.GetCompletionsAsync(this.ModelId, options, cancellationToken));
+            async () => await this.Client.GetCompletionsAsync(this.ModelId, options, cancellationToken).ConfigureAwait(false)).ConfigureAwait(false);
```

Comment:
Looks like a search/replace mistake... you only need one ConfigureAwait, not two :)